cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://dev.local:7000"
```

Comment:
This can be overridden by an env variable in the CI : https://docs.cypress.io/guides/guides/environment-variables#Setting.
This config file needs some documentation however !

Reply:
nice ! :+1: